src/ngraph/node.cpp

```diff
@@ -211,6 +235,15 @@ descriptor::Input& Node::get_input_descriptor(size_t position)
     return m_inputs.at(position);
 }
 
+const descriptor::Input& Node::get_input_descriptor(size_t position) const
+{
+    if (m_inputs.size() > position)
+    {
+        return m_inputs.at(position);
+    }
+    NODE_VALIDATION_CHECK(this, false, "Invalid input position", position);
```

Comment:
If you move this to line 240 and put the m_inputs.size() > position as the conditional then the code is much cleaner, two lines instead of 5